apps/web/src/app/style.css

```
@@ -1606,3 +1651,93 @@ h2 {
+.inline-name-edit:focus-visible {
+  opacity: 0.55;
+}
+.inline-name-edit:hover {
+  opacity: 1;
+}
+.inline-name-field {
+  font: inherit;
+  color: inherit;
+  padding: 0 0.2em;
+  border: 0;
+  border-bottom: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)